netlify/edge-functions/includes.mts

```diff
@@ -1,25 +1,26 @@
-import { HTMLRewriter, HTMLRewriterTypes } from 'https://ghuc.cc/worker-tools/html-rewriter/index.ts';
```

Comment:
Was getting a Deno error on this one. The library exports the `Element` type directly, so I used that.